catalog/transformer/catalog-transformer-xml/src/main/java/ddf/catalog/transformer/xml/adapter/AdaptedMetacard.java

```diff
@@ -68,10 +70,10 @@
 
   private Attribute idAttribute;
 
-  // Suppressing Warnings and using ArrayList rather than List here because
-  // ArrayList implements Serializable (List does not).
+  // Suppressing Warnings and using HashSet rather than Set here because
+  // HashSet implements Serializable (Set does not).
   @SuppressWarnings("all")
```

Comment:
✏️ This suppression could be more specific. Something about interface vs implementation declarations iirc.

Reply:
Sure, but given this is a straight forward port I'm going to leave it be for now.